audit/projects/kubernetes-public/secrets/slack-moderator-words-config/description.json

```diff
@@ -0,0 +1,10 @@
+{
+  "createTime": "2021-02-23T23:53:36.776896Z",
+  "labels": {
+    "app": "slack-infra"
+  },
+  "name": "projects/127754664067/secrets/slack-moderator-words-config",
```

Comment:
This was me and is expected, did this as part of https://github.com/kubernetes/k8s.io/pull/1696#issuecomment-784618406